README.md

```diff
@@ -31,38 +31,48 @@
 
 <br>
 
+Strapi is a free and open-source headless CMS delivering your content anywhere you need.
+
+- **Keep control over your data**. With Strapi, you know where your data is stored and you keep full control at any time.
+- **Self hosted**. You can host and scale Strapi project the way you want. You can choose any hosting platform you want: AWS, Netlify, Heroku at VPS or a dedicated server. You can scale as you grow, 100% independently.
+- **Database agnostic**. You can choose the database you prefer. Strapi works with SQL & NoSQL databases: MongoDB, PostgresQL, MySQL, SQLite.
+- **Customisable**. You can easily build your own logic by fully customising APIs, routes or plugins to perfectly fit your needs.
+
 ## Getting Started
 
 <a href="https://strapi.io/documentation/3.x.x/getting-started/quick-start.html" target="_blank">Read the Getting Started tutorial</a> or follow the steps below:
 
 #### 🖐 Requirements
 
 Operating system:
- * Ubuntu/Debian
- * CentOS/RHEL
- * MacOS
- * Windows - [WSL Only](https://github.com/strapi/strapi/wiki/Frequently-Asked-Questions#windows)
- * Docker - [Docker-Repo](https://github.com/strapi/strapi-docker)
+
+- Ubuntu/Debian
+- CentOS/RHEL
+- MacOS
+- Windows
+- Docker - [Docker-Repo](https://github.com/strapi/strapi-docker)
 
 **Please note that Strapi may work on other Operating Systems, but are not tested at this time.**
 
 Node:
- * NodeJS 10.x
- * NPM >= 6.x
+
+- NodeJS 10.x
+- NPM >= 6.x
 
 **Please note that right now Node 11 is not supported, and the current Node LTS (v10) should be used.**
```

Comment:
Propose: 
**Please note that right now Node 11 is not supported, and the current Node LTS (v10) is the official supported version.**

Reply:
I would change that to 11/12 as 11 should never be supported and support for 12 is planned I believe.